src/_statusline-semaphore.ts

```
@@ -0,0 +1,180 @@
+		catch: error => new Error(`Failed to check semaphore: ${String(error)}`),
+	});
+}
+
+/**
+ * Updates the semaphore with new execution data
+ */
+export function updateSemaphore(
+	sessionId: string,
+	output: string,
+): Result.Result<void, Error> {
+	return Result.try({
```

ESLint

[error] Unsafe call of a(n) `error` type typed value.

(ts/no-unsafe-call)